src/Lean/Meta/Basic.lean

```diff
@@ -1158,6 +1158,55 @@ def isInductivePredicate (declName : Name) : MetaM Bool := do
       | _ => return false
   | _ => return false
 
+/-
```

Comment:
It feels weird to have this function here. I will try to find a better place for it that avoids the unnecessary `Elab/Term.lean` dependency.
I will merge the PR as-is, and move the function to a different place.